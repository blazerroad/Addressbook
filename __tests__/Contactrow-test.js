import React from 'react';
import renderer from 'react-test-renderer';
import ContactRow from '../components/contact/contactRow';

test('renders correctly', () => {
  const tree = renderer.create(<ContactRow item={{id : "1", firstName: "name", lastName:"lastname",phoneNo:"123"}} />).toJSON();
  expect(tree).toMatchSnapshot();
});