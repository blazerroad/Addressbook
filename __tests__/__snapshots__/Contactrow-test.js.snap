// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "white",
      "borderRadius": 10,
      "elevation": 4,
      "margin": 10,
      "opacity": 1,
      "padding": 10,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.23,
      "shadowRadius": 2.62,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "borderRadius": 50,
            "height": 60,
            "justifyContent": "center",
            "marginRight": 10,
            "width": 60,
          },
          Object {
            "backgroundColor": "#ae2f92",
          },
        ]
      }
    >
      <Text>
        n
         
        l
      </Text>
    </View>
    <View>
      <Text
        style={
          Object {
            "fontSize": 15,
            "fontWeight": "bold",
            "paddingBottom": 5,
            "paddingTop": 5,
          }
        }
      >
        name
          
        lastname
      </Text>
      <Text>
        123
      </Text>
    </View>
  </View>
</View>
`;
